library/src/main/java/org/xmtp/android/library/Client.kt

```diff
@@ -552,6 +559,13 @@ class Client() {
         return runBlocking { query(Topic.contact(peerAddress)).envelopesList.size > 0 }
     }
 
+    fun canMessage(addresses: List<String>): Boolean {
+        return runBlocking {
+            libXMTPClient != null && !libXMTPClient!!.canMessage(addresses.map { it.lowercase() })
```

Comment:
@richardhuaaa my tests failed after I implemented this. I think this is another scenario where lowercase should be handled on the rust side.

Reply:
https://github.com/xmtp/libxmtp/pull/468